ApplicationPattern.yaml

```diff
@@ -5072,6 +5051,65 @@ paths:
                                           properties:
                                             request:
                                               type: string
+                                - description: 'generic response profile'
+                                  type: object
+                                  required:
+                                    - uuid
+                                    - profile-name
+                                    - response-profile-1-0:response-profile-pac
+                                  properties:
+                                    uuid:
+                                      type: string
+                                    profile-name:
+                                      type: string
+                                      enum:
+                                        - 'action-profile-1-0:PROFILE_NAME_TYPE_ACTION_PROFILE'
+                                        - 'file-profile-1-0:PROFILE_NAME_TYPE_FILE_PROFILE'
+                                        - 'integer-profile-1-0:PROFILE_NAME_TYPE_INTEGER_PROFILE'
+                                        - 'response-profile-1-0:PROFILE_NAME_TYPE_GENERIC_RESPONSE_PROFILE'
+                                        - 'string-profile-1-0:PROFILE_NAME_TYPE_STRING_PROFILE'
+                                    response-profile-1-0:response-profile-pac:
+                                      type: object
+                                      required:
+                                        - response-profile-capability
+                                        - response-profile-configuration
+                                      properties:
+                                        response-profile-capability:
+                                          type: object
+                                          required:
+                                            - operation-name
+                                            - field-name
+                                            - datatype
+                                          properties:
+                                            operation-name:
+                                              type: string
+                                            field-name:
+                                              type: object
+                                              minProperties: 1
+                                              maxProperties: 1
```

Comment:
Here, since the properties "concrete-field-name" and "field-name-reference" are non-required attributes, the restriction minProperties:1 and maxProperties:1 alone will allow a chance for any unknown property to be accepted by openApi validator.  Adding "additionalProperties: false" restriction would help in this situation to restrict entry of incorrect property in request-body

Reply:
Changed as proposed despite it is useless at a GET.

Reply:
Changed as proposed despite it is useless at a GET.